test-cases/performance/perf-search-best-throughput-config.yaml

```diff
@@ -0,0 +1,32 @@
+test_duration: 3000
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=30m -schema 'replication(factor=3)' -mode cql3 native -rate threads=50 -pop seq=1..30000000"
```

Comment:
c-s has also options for size of columns - shouldn't we increase it to be more realistic than default c-s setting?

Reply:
For that moment all stress tests are use standard c-s schema. 
For other size appropriate config files could be add in future.

Reply:
@aleksbykov as a followup PR we can have the stress commands as parameters to the job.
For example, @fgelcer just needed to test the max throughput under certain conditions of cache hit ratio.